### ISSUE TITLE
Include funtion getAll()

### ISSUE THREAD

**Issue by JanOschii** [jakearchibald/idb-keyval#140]
**My getAll() function**
```ts
export function getAll<T=any>(
  customStore = defaultGetStore()
): Promise<T[]> {
  return customStore( 'readonly', (store) => promisifyRequest(store.getAll()) )
}
```

**Why / Use Case**
I need to load the full table (10000 entries) at startup in the memory. If I use your function `entries()` it needs 20 secs.
So I changed my concept and now every value field is an object with a key and value property. So i can use getAll() and have all the entries (keys and values) in under 100ms.  **200times faster!**

I thought I share this expirence with you.

**Comment by jakearchibald** [comment]
This gives the same result as `value`, no?

**Comment by JanOschii** [comment]
My test has the same results for both functions, but getAll() is 200 times faster.
```txt
Reading 10.000 entries with values() needs 12.168ms
Reading 10.000 entries with getAll() needs     62ms
```
The function getAll() don't use cursors to iterate throught the table. The function values() is using the cursor.
```ts
export function values<T = any>(customStore = defaultGetStore()): Promise<T[]> {
  const items: T[] = [];
  return eachCursor(customStore, (cursor) => items.push(cursor.value)).then(
    () => items,
  );
}
```

**Comment by jakearchibald** [comment]
Yeah, I avoided getAll due to lack of support in IE and the old Edge engine. However, it seems fine to add these as performance enhancements.

I don't think these should be new methods, just performance enhancements to the existing methods.

**Comment by jakearchibald** [comment]
Although, if you're dealing with that amount of data, you might find it easier to work with https://github.com/jakearchibald/idb

**Comment by jakearchibald** [comment]
Could you test https://github.com/jakearchibald/idb-keyval/pull/141/files for me?

**Comment by JanOschii** [comment]
I would, but if I try to install the branch I get:
```
Could not install from "github.com:jakearchibald\idb-keyval.git#optimise-get-all" 
as it does not contain a package.json file
```
